kits/java/compile.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+echo 'Generate source list'
+find . -name '*.java' > sources.txt
+echo 'Compile source'
+javac -classpath ./lib/jackson-core-2.13.4.jar:./lib/jackson-databind-2.13.4.jar:./lib/jackson-annotations-2.13.4.jar -d ./build @sources.txt
```

Comment:
Question regarding java best practices. This is a maven project and the `pom.xml` even declares its dependencies. Why does this PR include the (almost 2MB worth of) `.jar` files of said dependencies?

Reply:
In first version supposed to compile project on the server and runtime dependency wouldn't work in that case.
A week ago it became clear that developer need to add only jar and py to submission.

Reply:
I see, but in the current version they can be removed, right?